### DISCUSSION TITLE
What does the p2 label represent?

### DISCUSSION THREAD

**Discussion by maafk** [aws/aws-cdk#15913] [Q&A] [answered]
I opened a PR and got the `p2` label added to it.

What does `p2` represent?

**Comment by peterwoodworth** [comment, marked as answer]
Hey @maafk,

The labels `p2` and `p1` are priority labels, with `p1` designating higher priority. 

Our team has a lot of issues and PRs to get through, so we have to put some things on the backlog. If your PR was labelled with `p2` then that means we aren't able to get to it immediately

We use +1s to help prioritize our work, and are happy to revaluate this issue based on community feedback. You can reach out to the cdk.dev community on [Slack](https://join.slack.com/t/cdk-dev/shared_invite/zt-mso6p56d-qJp7SOTBvMaQuDrx7R2wHg) to solicit support for reprioritization.

**Reply by maafk** [reply]
Thanks for the clarification!
